### PR TITLE
update scc_manager to 1.3.1

### PR DESCRIPTION
**Pull request by evgeni**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by evgeni** [comment]
this fails to run on rails5 though :(

**Comment by evgeni** [comment]
should be fixed as soon https://github.com/ATIX-AG/foreman_scc_manager/pull/11 is in a released gem

**Comment by evgeni** [comment]
updated to a version that is compatible with rails5 now

**Comment by evgeni** [comment]
@ekohl I have no idea what has to be done to make this also land in 1.17/3.6 repos (the scc manager there is broken at the moment, due to Rails5)

File: packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
-Version: 1.0.1
+Version: 1.3.1
 Release: 1%{?foremandist}%{?dist}
 Summary: Suse Customer Center plugin for Foreman
 Group: Applications/Systems
 License: GPL-3.0
 URL: https://www.orcharhino.com/
 Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
-Requires: foreman >= 1.13.0
+Requires: foreman >= 1.16.0
```
Comment (ekohl) [inline packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec]: Shouldn't this be 1.17 now since it's rails 5?

Comment (evgeni) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec]: Yeah, agreed. Bumped.

Comment (evgeni) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec]: also upstream: https://github.com/ATIX-AG/foreman_scc_manager/pull/13

**Comment by mdellweg** [comment]
@evgeni @ekohl Thank you!

**Pull request merged**